scripts/commands/update/update.js

```diff
@@ -16,27 +14,54 @@ const { ColorToken, GradientToken, SolidToken } = require("../../models");
 /**
  * Update all tokens within this project.
  */
-const update = () => {
+const update = (format) => {
   // Next tokens are mapped onto legacy tokens. These must be normalized to
   // match the existing local tokens.
-  const next = {
-    functional: new ColorToken({
-      format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
-      data: nextTokens.core["core color"],
-    }).normalize(),
-    decorative: new ColorToken({
-      format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
-      data: nextTokens.core["decorative color"],
-    }).normalize(),
-    gradation: new GradientToken({
-      format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
-      data: { gradation: nextTokens.core["gradation color"] }, // automated gradation token is missing the `gradation` key.
-    }).normalize(),
-    solids: new SolidToken({
-      format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
-      data: nextTokens.core["mobile solid background"],
-    }).normalize(),
-  };
+  let next;
+  const nextTokens = {};
+  if (format === ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED) {
+    nextTokens.abstract = require("momentum-abstract/color/core.json");
+    const automated = {
+      functional: new ColorToken({
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        data: nextTokens.abstract["core color"],
+      }).normalize(),
+      decorative: new ColorToken({
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        data: nextTokens.abstract["decorative color"],
+      }).normalize(),
+      gradation: new GradientToken({
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        data: { gradation: nextTokens.abstract["gradation color"] }, // automated gradation token is missing the `gradation` key.
+      }).normalize(),
+      solids: new SolidToken({
+        format: ColorToken.CONSTANTS.TOKEN_FORMATS.AUTOMATED,
+        data: nextTokens.abstract["mobile solid background"],
```

Comment:
not quite understanding why this is "mobile solid background" - why is this named / scoped "mobile"?